web/src/assets/locales/en.json

```diff
@@ -82,29 +82,29 @@
         },
         "allow_pr": {
           "allow": "Allow Pull Requests",
-          "desc": "Pipelines can run on pull requests."
+          "desc": "Allow pipelines to run on pull requests."
         },
         "allow_deploy": {
           "allow": "Allow deployments",
-          "desc": "Allow deployments from successful pipelines. Only use if you trust all users with push access."
+          "desc": "Permit 'deployment' runs for successful pipelines. All users with with push permissions can trigger these, so use with caution."
```

Comment:
> Permit 'deployment' runs for 

Im a bit nitpicking here, but if you add single quotes here other places would need single quotes a well, no?

Reply:
Agree. I think quotes are good here to mark the word as an event type more clearly and distinguish it from the "normal" meaning.